src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -105,8 +105,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     return Optional.empty();
   }
 
-  /** Creates and returns an Entity using parameters found in {@code request}. */
-  private Entity createLectureEntity(HttpServletRequest request) {
+  @VisibleForTesting
+  /** Creates and returns {@code lectureEntity} using parameters found in {@code request}. */
+  protected Entity createLectureEntity(HttpServletRequest request) {
     Optional<String> optionalLectureName = getParameter(request, NAME_INPUT);
     String lectureName = optionalLectureName.isPresent() ? optionalLectureName.get() : "";
```

Comment:
I think it might be wise to check that all of the params are present in the doGet/doPost, and `request.sendError(...)` if something's wrong.  I can't think of a reason you'd ever not want a lecture name/url, and allowing the request might lead to bugs/confusion later...

Reply:
This applies to all of the optionals below too, instead of defaulting to "" if missing, just send an error to the client

Reply:
Added as a TODO

Reply:
Lecture Name/URL are required as we required them to be filled in the form

Reply:
Technically someone can still submit a request without them...  For instance by using `fetch`.

Reply:
But TODO is fine for now